### PR TITLE
Pacifists now taunt secbots when attempting attacks with harm or disarm intent.

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

There exists an oversight where pacifists can attempt to attack Secbots such as Beepsky and there's no check for pacifism, only for the attack's intent. This causes Beepsky to get silently pissed off even though the player didn't attack them.

Instead of fixing this oversight, I've added additional feedback to turn this into a feature. Attacking Secbots as a pacifist in harm or disarm intent now taunts them instead. Beepsky will then vocally communicate to the security forces what's gotten him so rattled.

image

## Why It's Good For The Game

Secbots HATE pacifist scumbags and, when taunted, will chase them down without mercy. Gives a way for pacifists to distract security.

Turns an oversight into a minor feature.

Kinda fun meme PR?

Classic Timberpoes drunkcoding again?

## Changelog
:cl:
tweak: Pacifists can now taunt secbots such as Beepsky. Secbots absolutely loathe pacifist scumbags and will chase down any pacifist who taunts them, informing security in the process.
/:cl:

### CODE REVIEW

**Comment by Winterous** [comment]
>Secbots absolutely loate pacifist

loathe*

**Comment by Doosteroni** [comment]
lame, screw pacifists

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 1]
```diff
@@ -203,6 +203,15 @@ Auto Patrol: []"},
 		if(special_retaliate_after_attack(H))
 			return
 
+		// Turns an oversight into a feature. Beepsky will now announce when pacifists taunt him over sec comms.
+		if(HAS_TRAIT(H, TRAIT_PACIFISM))
+			H.visible_message("<span class='notice'>[H] taunts [src], daring them to give chase!</span>", \
+				"<span class='notice'>You taunt [src], daring them to chase you!</span>", "<span class='hear'>You hear someone shout a daring taunt!</span>", DEFAULT_MESSAGE_RANGE, H)
```
Comment (Jared-Fogle) [inline code/modules/mob/living/simple_animal/bot/secbot.dm:209]:
```suggestion
			H.visible_message("<span class='notice'>[H] taunts [src], daring [p_them()] to give chase!</span>", \
				"<span class='notice'>You taunt [src], daring [p_them()] to chase you!</span>", "<span class='hear'>You hear someone shout a daring taunt!</span>", DEFAULT_MESSAGE_RANGE, H)
```

ya lazy bum

Comment (Timberpoes) [reply code/modules/mob/living/simple_animal/bot/secbot.dm:209]: AAAAA I JUST COPY-PASTED!

Please forgive me Mothblock-senpai.

**Pull request merged**